### DISCUSSION TITLE
cmfa的UDP直连失败

### DISCUSSION THREAD

**Discussion by ForestL18** [MetaCubeX/ClashMetaForAndroid#236] [Q&A] [answered]
UDP直连会出现route ip+net: netlinkrib: permission denied的错误。
详情见：
https://github.com/MetaCubeX/mihomo/issues/1291#issue-2313657715

**Comment by Nine499** [comment]
同样，所有直连udp不通。
感觉像是内核的问题，因为其他第三方的 android meta 也有这个问题

我的崩坏星穹铁道打不开。

**Comment by Nine499** [comment]
tag Prerelease-alpha
https://github.com/MetaCubeX/ClashMetaForAndroid/commit/fa627f0da8f1d43ebf250e489f0619073365137d
依旧存在问题

**Comment by ForestL18** [comment, marked as answer]
最新内核已修复该问题
[10f8ba4](https://github.com/MetaCubeX/mihomo/commit/10f8ba44345e24f26ec63bab8919fc287de7196d)

**Reply by Nine499** [reply]
但是 CMFA 没更新

**Comment by Nine499** [comment]
bc8c24af3b3c9375df41ac9172494ea19dd17747

该版本成功修复。
